### DISCUSSION TITLE
Pourbaix diagrams

### DISCUSSION THREAD

**Discussion by sabualkaz** [reaktoro/reaktoro#279] [Ideas]
It would be great if there is a method to create pourbaix diagrams for a system.

**Comment by allanleal** [comment]
Hello @sabualkaz , thanks for your interesting suggestion. It would really be a great addition to Reaktoro. Pourbaix diagrams will be considered as a potential new capability in the future when seeking funding for continual development of Reaktoro. If anyone is interested in helping with the development of this feature, please get in touch.

**Reply by sabualkaz** [reply]
I am looking forward to seeing how this can be incorporated into Reaktoro.
